wordpress-mobile/WordPress-iOS
# Add extra attributes to images on Aztec.

Fixes #7537 

To test:
 - Create a new post
 - Add an image to it 
 - Wait for upload to finish
 - Switch to HTML mode
 - See if width, height and wp-image-XX properties are applied to img tag.

Needs review: @elibud
